contents/javascript/Javascript-promises.md

```diff
@@ -12,7 +12,7 @@
 
 # Javascript: Promises
 
-Author: Daniel Berzin Chua
```

Comment:
I suggest using italics when a term is introduced first. e.g., synchronous, asynchronous, callback hell, etc.